include/swift/SILOptimizer/Analysis/ArraySemantic.h

```diff
@@ -131,6 +135,11 @@ class ArraySemanticsCall {
   /// Returns true on success, false otherwise.
   bool replaceByValue(SILValue V);
 
+  /// Replace a call to append(contentsOf: ) with a series of
+  /// append(element: ) calls.
+  bool replaceByAppendingValues(SILModule &M, SILFunction* AppendFn,
+    llvm::SmallVectorImpl<SILValue> &Vs, ArrayRef<Substitution> Subs);
```

Comment:
Can you not abbreviate Vs? This is generally not an abbreviation that we use. I would suggest Vals or Values.